packages/api-key-service/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "api-key",
+  "dependencies": {
+    "@aws/dynamodb-data-mapper-annotations": "^0.7.3",
+    "@homeservenow/serverless-aws-handler": "^0.2.4",
+    "class-transformer": "^0.4.0",
+    "class-validator": "^0.13.1"
+  },
+  "scripts": {
```

Comment:
Will expect a build, release and publish script too - you can echo skip them obvs, just makes the pipeline simple ie "run these tasks in order"